mlflow/ml-package-versions.yml

```diff
@@ -95,9 +98,11 @@ keras:
     minimum: "2.3.0"
     maximum: "2.8.0"
     requirements:
-      "<= 2.3.1": ["tensorflow==2.2.1", "scikit-learn", "pyspark", "pyarrow", "transformers<=4.11.3"]
-      "> 2.3.1, < 2.6.0": ["tensorflow<2.5.0", "scikit-learn", "pyspark", "pyarrow", "transformers"]
-      ">= 2.6.0": ["tensorflow", "scikit-learn", "pyspark", "pyarrow", "transformers"]
+      ">= 0.0.0": ["scikit-learn", "pyspark", "pyarrow"]
+      "<= 2.3.1": ["tensorflow==2.2.1", "transformers<=4.11.3"]
+      "> 2.3.1, < 2.6.0": ["tensorflow<2.5.0", "transformers"]
+      "< 2.6.0": ["protobuf<4.0.0"]
```

Comment:
Is this redundant ?
why not writing `"< 4.0.0": ["protobuf"]` ?

Reply:
It's not redundant. `"< 2.6.0": ["protobuf<4.0.0"]` means install `protobuf<4.0.0` if keras version is `< 2.6.0`.